crates/control/src/behavior/stand_up.rs

```diff
@@ -4,7 +4,10 @@ use types::{
     world_state::WorldState,
 };
 
-pub fn execute(world_state: &WorldState) -> Option<MotionCommand> {
+pub fn execute(world_state: &WorldState, maximum_standup_attempts: u32) -> Option<MotionCommand> {
+    if world_state.robot.stand_up_count >= maximum_standup_attempts {
```

Comment:
The counter increments the moment we begin the standup motion.
If `maximum_standup_attempts` is set to 1, we'll give up on the first attempt, right?
I think this should be just `>`.